pkg/controller/host/validation.go

```diff
@@ -86,7 +86,7 @@ func (r *Reconciler) validate(host *api.Host) error {
 			Status:   True,
 			Reason:   Completed,
 			Category: Advisory,
-			Message:  "The host has been validated.",
+			Message:  "The host resources have been validated.",
```

Comment:
it was suggested in the bug to elaborate on what's being validated (Host inventory reference, IP address, and credentials) - I'm not sure the term 'host resources' would be clear to users